db/migrate/20190403081352_add_bmap_flowcell_table.rb

```diff
@@ -2,7 +2,6 @@
 
 class AddBmapFlowcellTable < ActiveRecord::Migration
   def change
-    # rubocop:disable Rails/CreateTableWithTimestamps
     create_table :bmap_flowcell, primary_key: :id_bmap_flowcell_tmp do |t|
```

Comment:
We still disable these cops, but just do so globally. Avoids an unwitting developer getting confused by Rubocop insisting on something we don't actually want.